src/frontend/src/app/myprofile/myprofile.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { PlayerData } from '../types/PlayerData';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-myprofile',
+  templateUrl: './myprofile.component.html',
+  styleUrls: ['./myprofile.component.scss'],
+})
+export class MyprofileComponent {
+  id: string = '';
+  players: PlayerData[] = [];
```

Comment:
There'll only be one player here, so no need to use a list.

Reply:
Removed list